src/main/java/com/splunk/hecclient/HecAckPoller.java

```diff
@@ -316,7 +316,7 @@ private void findAndRemoveTimedoutBatches(Map<Long, EventBatch> batches, List<Ev
     }
 
     private void handleAckPollResponse(String resp, HecChannel channel) {
-        log.debug("ackPollResponse={}, channel={}", resp, channel);
+        log.debug("channel={}", channel);
```

Comment:
```suggestion
        log.debug("Ack response for channel={}", channel);
```
Let's add a little context to the log. Is this apt?